lib/components/chat_history/twitch/message_link_preview.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:metadata_fetch/metadata_fetch.dart';
 import 'package:flutter/material.dart';
 
+Map<String, _TwitchClipData> linkPreviewCache = {};
```

Comment:
Can you remove the link preview change from this PR?

Reply:
I copied and pasted the file `message_link_preview.dart` from main to my local one, and pushed it.